changelog.rst

```diff
@@ -674,3 +674,4 @@ Improvements:
 .. _`tk`: https://github.com/kanet77
 .. _`Owen Stephens`: https://github.com/owst
 .. _`Russell Davies`: https://github.com/russelldavies
+.. _`Dick Marinus`: https://github.com/meeuw
```

Comment:
Thanks for fixing this! Maybe a remark should be added to the PR template to add your name twice to the changelog?